starboard/BUILD.gn

```diff
@@ -16,7 +16,7 @@ import("//starboard/build/config/os_definitions.gni")
 import("//starboard/build/config/starboard_target_type.gni")
 import("//starboard/contrib/cast/cast.gni")
 
-if (sb_is_modular) {
+if (sb_is_modular && false) {
```

Comment:
@niranjanyardi this is something I included because the libcxx deps weren't there when I initially was working on this—are you fine with me leaving it and you removing it at the same time as you get those dependencies to actually work? Otherwise I can potentially fix this in a day or two, would just be a little more time to get this merged

Reply:
Adding a TODO would be good https://b.corp.google.com/issues/371241293 . I'm okay with leaving the false